ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/CancunTargetingGasLimitCalculatorTest.java

```diff
@@ -187,7 +188,7 @@ void shouldCalculateCorrectlyPragueBlobGasPerBlob() {
   public void shouldCalculateOsakaExcessBlobGasCorrectly(
       final long parentExcess, final long used, final long expected) {
     final long usedBlobGas = osakaGasCalculator.blobGasCost(used);
-    assertThat(osakaGasLimitCalculator.computeExcessBlobGas(parentExcess, usedBlobGas))
+    assertThat(osakaGasLimitCalculator.computeExcessBlobGas(parentExcess, usedBlobGas, 0L))
```

Comment:
This is using `CancunTargetingGasLimitCalculator`. Should it be `OsakaTargetingGasLimitCalculator`?